proto/cmp/services/accommodation/v1alpha/property_types.proto

```diff
@@ -100,6 +98,13 @@ enum CategoryUnit {
   CATEGORY_UNIT_PALMS = 2;
 }
 
+enum Status {
```

Comment:
Correct!

Reply:
Now we have two `status` fields, one in `Property` and one in `PropertyExtendedInfo` (which is linked to the `Property` type). I think we need to remove the one in the extended info message, because it will already be included in the `Property`. What do you think?